src/pages/PokemonList.jsx

```diff
@@ -92,52 +92,46 @@ export const PokemonList = () => {
 		return () => clearTimeout(timer);
 	}, [pokemon.search]);
 
-	return (<>
-		<div className={'pokemon-list-panel'}>
-			<div className={'left'}>
-				<div className={'pokemon-name'}>
-					{pokemon.select ? uppercaseFirstLetter(pokemon.select) : 'Select a pokemon!'}
-				</div>
-				<div className={'pokemon-artwork-holder'}>
-					{pokemon.select ? <img src={pokemon.officialArtwork} alt={'official artwork'}/> : <></>}
+	return (
+		<>
+			<div className={'pokemon-list-panel'}>
+				<div className={'left'}>
+					<div className={'pokemon-name'}>
+						{pokemon.select ? uppercaseFirstLetter(pokemon.select) : 'Select a pokemon!'}
+					</div>
+					<div className={'pokemon-artwork-holder'}>
+						{pokemon.select ? <img src={pokemon.officialArtwork} alt={'official artwork'}/> : <></>}
+					</div>
+					<div className={'pokemon-view-details-button-holder'}>
+						{pokemon.officialArtwork !== pokeballLoadingImage
+                            && <button onClick={toggleViewDetails}>View
+                                details</button>}
+					</div>
 				</div>
-				<div className={'pokemon-view-details-button-holder'}>
-					{pokemon.officialArtwork !== pokeballLoadingImage
-                        && <button onClick={toggleViewDetails}>View
-                            details</button>}
+				<div className={'right'}>
+					<input type={'search'} className={'search-bar'} value={pokemon.search}
```

Comment:
These elements are directly trying to fetch data according to the pokemon selected name.